src/cnlpt/train_system.py

```diff
@@ -263,17 +271,19 @@ def main(
             config.finetuning_task = data_args.task_name
             config.relations = relations
             config.tagger = tagger
```

Comment:
I wonder if these should be inside the if statement as well? Or whether we want to make sure there's not a conflict between the values assigned based on the task and the existing task in the config (if --keep-existing-classifiers is enabled)

Reply:
I agree I will move them into if statement and add sanity checking codes!